src/donejs-carousel-plugin.js

```diff
@@ -606,53 +600,55 @@ export const ViewModel = DefineMap.extend({
 	* @function makeAllSlidesOpaque
 	*
 	* @description
-	* Reset the opacity of all the slides when breaking on Desktop
+	* Reset the opacity of all the slides when breaking on desktop
 	*
 	*/
 	makeAllSlidesOpaque() {
 		let classSelector = this.classSelector;
 
 		$(`${classSelector} .slide`).css({'opacity': 1});
+
 	},
 	/**
-	* @function makeAllSlidesVisible
+	* @function makeOnlyActiveSlideOpaque
 	*
 	* @description
-	* The visibility of slides allows links to function properly
-	* Make all slides visible (only during transitions when fading)
+	* Make only the active slide opaque and all other slides should be transparent
 	*
 	*/
-	makeAllSlidesVisible() {
+	makeOnlyActiveSlideOpaque() {
 		let classSelector = this.classSelector;
 
-		$(`${classSelector} .slide`).css({'visibility': 'visible'});
+		$(`${classSelector} .slide`).css({'opacity': 0});
+		$(`${classSelector} .slide.active`).css({'opacity': 1});
 	},
 	/**
-	* @function makeOnlyActiveSlideOpaque
+	* @function resetZIndexAllSlides
 	*
 	* @description
-	* Make only the active slide opaque and all other slides should be transparent
+	* Reset the z-index of slides when resizing the window and going from
+	* broken on desktop to mobile view
 	*
 	*/
-	makeOnlyActiveSlideOpaque() {
+	resetZIndexAllSlides() {
 		let classSelector = this.classSelector;
 
-		$(`${classSelector} .slide`).css({'opacity': 0});
-		$(`${classSelector} .slide.active`).css({'opacity': 1});
+		$(`${classSelector} .slide`).css({'z-index': 0});
 	},
+
 	/**
-	* @function makeOnlyActiveSlideVisible
+	* @function setActiveSlideZIndex
 	*
 	* @description
-	* The visibility of slides allows links to function properly
-	* Make only the active slide visible and all other slides should be hidden
+	* Setting a larger z-index for the active slide allows the links to function properly
+	* Set z-index to 100 for active slide, 0 for all other slides
 	*
 	*/
-	makeOnlyActiveSlideVisible() {
+	setActiveSlideZIndex() {
 		let classSelector = this.classSelector;
 
-		$(`${classSelector} .slide`).css({'visibility': 'hidden'});
-		$(`${classSelector} .slide.active`).css({'visibility': 'visible'});
+		$(`${classSelector} .slide`).css({'z-index': 0});
```

Comment:
@kevcom : i assume you can't just remove existing `z-index`-property?

Reply:
i probably can. good suggestion.

Reply:
apparently all the examples say to just set the css properties to `''`. there's apparently no clean way to completely remove the property in jQuery or javascript. But also for code readability probably makes sense to keep it the way it is.